lib/Services/delorean-es.js

```diff
@@ -106,6 +106,29 @@ class DeloreanES {
     return this.auth.send(this.service, 'GET', url, { query });
   }
 
+  /**
+   * @param {String}  userId
+   * @param {String}  deviceId
+   * @param {Object}  query
+   */
+  getUnits(userId, deviceId, query) {
+    const url = `${this.url}/devices/${deviceId}/readings/units`;
+    query = { ...query, user_id: userId };
+    return this.auth.send(this.service, 'GET', url, { query });
+  }
+
+  /**
+   * @param {String}  userId
+   * @param {String}  deviceId
+   * @param {Object}  query
+   * @param {Object}  stream
+   */
+  getDeviceReportStream(userId, deviceId, query, stream) {
+    const url = `${this.url}/devices/${deviceId}/stream/report`;
+    query = { ...query, user_id: userId };
+    return this.auth.send(this.service, 'GET', url, { query, stream }, 0);
```

Comment:
As response timeout was being increased, I suggest adding an opts variable to method signature and pass that through send function to allow higher timeouts for streams.

And `0` is not required, this is only to be used for authorization failure fallback.

```js
getDeviceReportStream(userId, deviceId, query, opts) {
  // stream, response, and deadline will be passed through and override default timeouts
  return this.auth.send(this.service, 'GET', url, { query, ...opts });
}
```

Reply:
Only ES stream need a longer timeout. Let's keep this for now and if we need it in another place, let's change it.